### PR TITLE
Ukds 52

### PR DESCRIPTION
**Pull request by jameyhart**
Add Available Directories endpoint and a couple of minor fixes.

### CODE REVIEW

File: api/serverless.yml [hunk 1]
```diff
@@ -31,6 +31,14 @@ provider:
               - 'lambda:InvokeFunction'
           Resource: '*'
 functions:
+  availableDirectories:
+    handler: src/routes/postcodes.getAvailableDirectories
+    warmup: true
+    events:
+      - http:
+          path: /postcodes/available-directories
+          method: GET
```
Comment (davidplumb) [inline api/serverless.yml:40]: Maybe change 'available-directories' to 'directories' as the available bit is implied if they are returned (more RESTfull I think)

Comment (jameyhart) [reply api/serverless.yml:40]: Yes, I think you're right.
I'll look at this now.

**Review by davidplumb — APPROVED** [review]
Changes look good, I'm fine with merging the changes

**Review by nathan-at-jisc — APPROVED** [review]
lgtm

**Pull request merged**